test_shopcart.py

```diff
@@ -28,6 +28,11 @@ def test_index(self):
         self.assertEqual( resp.status_code, status.HTTP_200_OK )
         self.assertTrue ('Shopcart Demo REST API Service' in resp.data)
 
+    def test_list_shopcarts(self):
+        resp = self.app.get('/shopcarts')
+        self.assertEqual( resp.status_code, status.HTTP_200_OK )
+        self.assertTrue( len(resp.data) > 0 )
```

Comment:
I think it depends how we populate our memory (or database). If we decide to put something in the shopcarts before the test, I think this test is ok. Although we can assert more information if we already know it.